### DISCUSSION TITLE
Loki: Render empty labels in legend as empty strings

### DISCUSSION THREAD

**Discussion by gmintoco** [grafana/grafana#45758] [Dashboards, Panels & Visualizations]
As in this PR for Prometheus: https://github.com/grafana/grafana/pull/22355

Loki dashboards should render labels that are in legend {{label_here}} but missing from the data, as empty strings.

Prevents issues like this - which are hard to resolve with lokis query syntax (no label replace tools etc.): 
![image](https://user-images.githubusercontent.com/50625416/155263000-0dff3c8e-2963-41f2-adbc-8242cd51b067.png)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
